src/couchapps/WorkQueue/views/elementsDetailByWorkflowAndStatus/reduce.js

```diff
@@ -1,17 +0,0 @@
-function (key, values, rereduce) {
```

Comment:
Since you deleted this reduce function, shouldn't you also rollback the rewrite arg:
https://github.com/dmwm/WMCore/blob/master/src/couchapps/WorkQueue/rewrites.json#L7
?